ui/components/ui/disclosure/disclosure.scss

```diff
@@ -7,8 +7,9 @@
     padding-left: 24px;
     padding-bottom: 10px;
     font-weight: bold;
+    display: flex;
 
-    i {
+    &--icon {
```

Comment:
@georgewrmarshall Am I correct in that we want to remove these styles completely and instead add `marginInlineStart={x}` and `marginInlineEnd={x}`?

Reply:
We can add the `maginInlineStart={-5}` and that should work but I am not sure about using `marginInlineEnd` since its value is `10 `here.

Reply:
We can just round down to the closest divisible by 4 value so `8px` `marginInlineEnd={2}` I don't think the box component supports negative values though so `-5` won't work. Do you know what the reason is for the negative margin? It seems quite random?

Reply:
Thanks @georgewrmarshall @darkwing . I have updated the marginInlineEnd value. Seems like the negative margin was added to keep the icon in place, since `padding-left: 24px;` in `disclosure-summary` was adding extra padding to the container.

### New
![Screenshot 2023-03-06 at 3 40 55 PM](https://user-images.githubusercontent.com/39872794/223080516-28d05ac6-407a-43e8-a3c7-39f9621d0685.jpg)